libcxx/include/limits

```diff
@@ -137,9 +137,9 @@ protected:
   typedef _Tp type;
 
   static _LIBCPP_CONSTEXPR const bool is_specialized = false;
-  _LIBCPP_HIDE_FROM_ABI static _LIBCPP_CONSTEXPR type min() _NOEXCEPT { return type(); }
-  _LIBCPP_HIDE_FROM_ABI static _LIBCPP_CONSTEXPR type max() _NOEXCEPT { return type(); }
-  _LIBCPP_HIDE_FROM_ABI static _LIBCPP_CONSTEXPR type lowest() _NOEXCEPT { return type(); }
+  _LIBCPP_NODISCARD _LIBCPP_HIDE_FROM_ABI static _LIBCPP_CONSTEXPR type min() _NOEXCEPT { return type(); }
```

Comment:
These should all be `_LIBCPP_NODISCARD_EXT`. Please also add test similar to `libcxx/test/libcxx/diagnostics/bit.nodiscard_extensions.compile.pass.cpp` and `libcxx/test/libcxx/diagnostics/bit.nodiscard_extensions.verify.cpp`.

Reply:
Done.